index.js

```diff
@@ -0,0 +1,97 @@
+document.addEventListener("DOMContentLoaded",() => {
+    let getPokeButton = document.querySelector(".getPokeButton")
+    let battleButton = document.querySelector(".battleButton")
+    
+    
+    const randomNum = (min,max) => {
+        return Math.floor(Math.random() * (max - min + 1))
+    }
+    const getPokemon  = async () => {
+       
+        try{ 
+        let data = document.querySelector(".data")
+
+            
+           
+       let pokemon1 = await axios.get(`https://pokeapi.co/api/v2/pokemon/${randomNum(1,89)}`)
+       let pokemon2 = await axios.get(`https://pokeapi.co/api/v2/pokemon/${randomNum(1,89)}`)
+       
+       
+      
+       let pokemon1ul = document.querySelector("#pokemon1ul")
```

Comment:
You can make this function take in an argument and let it be the div you cant to append to